## Patch originally from Fedora vnc-4.1.2-21.fc8 RPM - vnc-64bit.patch

--- vnc-4_1_1-unixsrc/unix/xc/programs/Xserver/vnc/XserverDesktop.h.64bit	2005-03-11 15:08:41.000000000 +0000
+++ vnc-4_1_1-unixsrc/unix/xc/programs/Xserver/vnc/XserverDesktop.h	2005-12-06 14:31:11.000000000 +0000
@@ -29,6 +29,8 @@
 #include <rfb/VNCServerST.h>
 #include <rdr/SubstitutingInStream.h>
 
+#include "xorg-server.h"
+
 extern "C" {
 #define class c_class;
 #include <scrnintstr.h>
--- vnc-4_1_1-unixsrc/unix/xc/programs/Xserver/vnc/Xvnc/xvnc.cc.64bit	2005-12-06 14:31:24.000000000 +0000
+++ vnc-4_1_1-unixsrc/unix/xc/programs/Xserver/vnc/Xvnc/xvnc.cc	2005-12-06 14:31:30.000000000 +0000
@@ -973,25 +973,6 @@
 
 } /* end InitOutput */
 
-#ifdef DPMSExtension
-extern "C" {
-#if NeedFunctionPrototypes
-  void DPMSSet(CARD16 level)
-#else
-    void DPMSSet(level)
-    CARD16 level;
-#endif
-  {
-    return;
-  }
-
-  Bool DPMSSupported()
-  {
-    return FALSE;
-  }
-}
-#endif
-
 /* this is just to get the server to link on AIX */
 #ifdef AIXV3
 int SelectWaitTime = 10000; /* usec */
